packages/lib/JoplinServerApi.ts

```
@@ -177,6 +177,7 @@ export default class JoplinServerApi {
 
 		if (sessionId) headers['X-API-AUTH'] = sessionId;
 		headers['X-API-MIN-VERSION'] = '2.6.0'; // Need server 2.6 for new lock support
+        if (!headers['Accept']) headers['Accept'] = 'application/json';
```

ESLint

[error] Expected indentation of 2 tabs but found 8 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 2 tabs but found 8 spaces.

(indent)